schema/common/validation/epiphany-cluster.yml

```diff
@@ -122,7 +122,7 @@ properties:
         default: ''
         examples:
         - extension
-        pattern: "^(.*)$"
+        pattern: "^[a-zA-Z0-9.-]{0,20}$"
```

Comment:
Do we support capital letters in hostname? https://github.com/epiphany-platform/epiphany/blob/dd972cda8ff0f77cf609d7e7051cf66dfe3768d8/schema/any/validation/infrastructure/machine.yml#L12

Reply:
Technically this is a domain extension, as in somehostname.thispart. I have seen from time to time the use of capitals there but don't think its part of official spec.

Reply:
So we leave it as it is?

Reply:
Also Azure does not have not have any issues with this: 
![image](https://user-images.githubusercontent.com/11056665/147650068-07494769-3473-46dd-8613-865bd924000a.png)

Reply:
For me we can leave it as is.

Reply:
Yes, this is because that validation is for 'any' provider.